spec/api/v1/sparql_spec.rb

```diff
@@ -0,0 +1,89 @@
+require_relative 'shared_examples/missing_envelope'
+require_relative 'shared_examples/signed_endpoint'
+require 'query_log'
+
+RSpec.describe API::V1::Sparql do
+  before { VCR.turn_off! }
```

Comment:
I believe VCR needs to be turned back on in an `after` block.

Reply:
Indeed. I've fixed this.